doc/conf/Doxyfile.in

```diff
@@ -1,27 +1,29 @@
 QUIET                  = YES
 
 # General project info
-PROJECT_NAME           = "Ginkgo"
-PROJECT_NUMBER         = "@PROJECT_VERSION_TAG@ @PROJECT_VERSION@"
-PROJECT_BRIEF          = "Ginkgo is a numerical linear algebra library targeting manycore architectures."
+PROJECT_NAME           = "@PROJECT_NAME@" 
+PROJECT_NUMBER         = "Generated from @GINKGO_GIT_BRANCH@ branch based on @PROJECT_VERSION_TAG@. Ginkgo version @PROJECT_VERSION@ "
+PROJECT_BRIEF          = "@PROJECT_DESCRIPTION@" 
 INLINE_INHERITED_MEMB  = NO
 
 # Paths and in/exclusion patterns
-INPUT                  = @DIR_BASE@/core @DIR_BASE@/omp @DIR_BASE@/cuda @DIR_BASE@/reference @MAINPAGE@
-INCLUDE_PATH           = @DIR_BASE@/include @DIR_BASE@
+#INPUT                  = See the specific doxy files for usr and dev.  
+INCLUDE_PATH           = @DIR_BASE@/include @DIR_BASE@ 
 OUTPUT_DIRECTORY       = @DIR_OUT@
-EXAMPLE_PATH           = @DIR_BASE@/examples
+EXAMPLE_PATH           = @DIR_BASE/doc/examples@
```

Comment:
Does this really work ? The way this is put it's like there is a CMake variable named `DIR_BASE/doc/examples`. Maybe you should stick the `@` to the `DIR_BASE`.

Reply:
Sorry, that was a typo.